features/lwipstack/LWIPInterface.cpp

```diff
@@ -457,6 +495,30 @@ nsapi_error_t LWIP::add_l3ip_interface(L3IP &l3ip, bool default_if, OnboardNetwo
 nsapi_error_t LWIP::remove_l3ip_interface(OnboardNetworkStack::Interface **interface_out)
```

Comment:
Documentation for remove_l3ip_interface is incorrect in LWIPStack.h:

```
    /** Remove a network interface from IP stack
     *
     * Connects L3IP layer with the IP stack and initializes all the required infrastructure.
     * This function should be called only once for each available interface.
```

Reply:
done